fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/AbstractEventCreationFragment.java

```diff
@@ -55,33 +55,80 @@ public abstract class AbstractEventCreationFragment extends Fragment {
   private EditText startTimeEditText;
   private EditText endTimeEditText;
 
+  private boolean toBeCleared = true;
+
   public void setDateAndTimeView(View view) {
+    long currentMillis = System.currentTimeMillis();
+    long suggestedEndMillis = currentMillis + 1000 * 60 * 60; // Adding one hour
```

Comment:
is it interesting to add a class constant ?

Reply:
I updated it